vigetlabs/wordpress-site-starter
# [#130] Block clean up and refactor

# Summary

- Adding some nice defaulting settings
- Setting widths for all blocks
- Updated the default setting of all the blocks so you can set a bg color and it is full width, but the content is constrained. (does not apply to the alert banner)
- Fixing light text with a dark background on a few blocks
- Updating the page template - removing block spacer and adding padding
- Removing the simple block patterns and putting them in the render  `template` array. 
- Adding glob to load CSS files with `./**/*.css`
- Updating all the plop templates to match our new file structure. 

## Issues

* #130 

## Testing Instructions

1. Add a block to a page (any of our custom ones but the alert banner).
2. Set a bg color and make sure it is full width but the content is constrained. 
3. Make sure you can save/update blocks without any errors. 
4. Use plop `ddev npm run plop block` and add a new block. 

## Screenshots
### New file structure. 
<img width="224" alt="Screenshot 2024-08-14 at 10 02 07 AM" src="https://github.com/user-attachments/assets/62c096c2-1f88-4f14-afe7-50cb3bb204bb">

### Blocks now default to full-width bg color.
![wpstarter ddev site_this-is-a-link_sample-page_](https://github.com/user-attachments/assets/469acf1c-ac51-4aa6-a6d6-968f6e79df69)
